docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -611,6 +611,119 @@ JSON::
 }
 ----
 ====
+NOTE: It is possible add multiple data sources.
+
+[tabs]
+====
+YAML::
++
+[source,yaml]
+----
+sources:
+  - policy:
+      - git::https://github.com/enterprise-contract/ec-policies.git//policy
+    data:
+      - oci::quay.io/lucarval/policy-data:latest
+      - oci::quay.io/lucarval/additional-policy-data:latest
+    ruleData:
+      rule_data_custom:
+        allowed_registry_prefixes:
+          - trusted-registry.io/trusted-images/
+          - docker.io/acme-company/
+----
+JSON::
++
+[source,json]
+----
+{
+  "sources": [
+    {
+      "policy": ["git::https://github.com/enterprise-contract/ec-policies.git//policy"],
+      "data": [
+        "oci::quay.io/lucarval/policy-data:latest",
+        "oci::quay.io/lucarval/additional-policy-data:latest"
+      ]
+      "ruleData": {
+        "rule_data_custom": {
+          "allowed_registry_prefixes": [
+            "trusted-registry.io/trusted-images/",
+            "docker.io/acme-company/"
+          ]
+        }
+      }
+    }
+  ]
+}
+----
+====
+
+WARNING: There must not be any duplicate array entries in the data sources. If there are, it will result in a merge error and failure.
```

Comment:
Does it have to say array or can it be any duplicate entries?

Reply:
I can change it to entries.